uobtheatre/bookings/test/test_mutations.py

```diff
@@ -417,17 +408,19 @@ def test_create_booking_with_new_taget_user(gql_client):
     psg = PerformanceSeatingFactory()
     request = """
         mutation {
-          createBooking(
-            performanceId: "%s"
-            targetUserEmail: "abc@email.com"
+          booking(
+              input: {
+                performance: "%s"
+                userEmail: "abc@email.com"
+              }
```

Comment:
This is the relay mutation standard. Not sure why but it is done at graphene level

Reply:
Yeah, I'm fairly sure this cropped up before and I disabled it as its feel unnecessary for us. Either way, we should be consistent with either having it or not having it. Do you think it's possible to remove it from the forms?

Reply:
I don't think you disabled it - it is currently used for all production/performance/concession type/discount mutations. I think it is fairly nice in being explicit and allows us to differentiate between updating model attributes and generally passing in other config arguments and the like

Reply:
From what I can tell it's not in any mutations other than those handled by the custom form stuff. Differentiation is a nice idea but not something we do, so only adds more complexity.

Reply:
It's not the custom form stuff that is doing it, it is grapheme properly implementing the relay spec. I don't think it is something we should change, and even if we did it would probably require a lot of overriding of package code which isn't really maintainable.  Plus I still see the benefits of this approach, especially wrt separating model data from other augments.

Reply:
Ah I think the issue is every other mutation doesn't use the relay standard (to avoid having the input nonsense). Either way we should stick to either relay or not...